lib/Directives/Contents.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\RST\Directives;
+
+use Doctrine\RST\Nodes\Node;
+use Doctrine\RST\Parser;
+
+final class Contents extends Directive
+{
+    public function __construct()
+    {
+    }
```

Comment:
:thinking: what is the point of that empty constructor?

Reply:
Leftover. Removed